walltaker.go

```diff
@@ -114,6 +115,17 @@ func main() {
 		log.Fatal(err)
 	}
 
+    bg, err := wallpaper.Get()
+    fmt.Println("Detected original wallpaper as: ", bg, "\r\n")
```

Comment:
```suggestion
    fmt.Println("Detected original wallpaper as: ", bg)
```

fmt.Println arg list ends with redundant newline; please remove :smiley_cat: